lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -54,12 +56,18 @@ def test_mc_dl1ab():
     assert os.path.exists(output_file)
 
 
+@pytest.mark.run(after='test_mc_dl1ab')
+def test_mc_dl1ab_validity():
+    dl1 = pd.read_hdf(os.path.join(output_dir, 'dl1_gamma_test_large.simtel.h5'), key=dl1_params_lstcam_key)
+    dl1ab = pd.read_hdf(os.path.join(output_dir, 'dl1ab.h5'), key=dl1_params_lstcam_key)
+    np.testing.assert_allclose(dl1, dl1ab, rtol=1e-4)
+
+
 @pytest.mark.run(after='test_lstchain_dl1_to_dl2')
 def test_mc_r0_to_dl2():
-    cmd = f'lstchain_mc_r0_to_dl2 -f {mc_gamma_testfile} -p {output_dir} -s1 False -o {output_dir}'
+    cmd = f'lstchain_mc_r0_to_dl2 -f {mc_gamma_testfile} -p {output_dir} -s1 True -o {output_dir}'
```

Comment:
why is the DL1 file now written?

Reply:
To be used in `test_mc_dl1ab_validity` to be checked vs `dl1ab.h5`.